lib/ansible/modules/tacp/tacp_info.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Lenovo
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.tacp_ansible import tacp_utils
+
+
+import tacp
+from tacp.rest import ApiException
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: tacp_info
+
+short_description: This is my test module
+
+version_added: "2.9"
+
+description:
+    - "This is my longer description explaining my test module"
+
+options:
+    name:
+        description:
+            - This is the message to send to the test module
+        required: true
+    new:
+        description:
+            - Control to demo if the result of this module is changed or not
+        required: false
+
+extends_documentation_fragment:
+    - tacp
+
+author:
+    - Xander Madsen (@xmadsen)
+'''
+
+EXAMPLES = '''
+# Pass in a message
+- name: Test with a message
+  tacp_instance:
+    name: hello world
+
+# pass in a message and have changed true
+- name: Test with a message and changed output
+  tacp_instance:
+    name: hello world
+    new: true
+
+# fail the module
+- name: Test failure of the module
+  tacp_instance:
+    name: fail me
+'''
+
+RETURN = '''
+original_message:
+    description: The original name param that was passed in
+    type: str
+    returned: always
+message:
+    description: The output message that the test module generates
+    type: str
+    returned: always
+'''
+
+
+def run_module():
+    # define available arguments/parameters a user can pass to the module
+    module_args = {
+        "api_key": {"type": 'str', "required": True},
+        "resource": {"type": 'str', "required": True,
+                     "choices": [
+                         "application",
+                         "application_group",
+                         "category",
+                         "datacenter",
+                         "firewall_profile",
+                         "instance",
+                         "marketplace_template",
+                         "migration_zone",
+                         "site",
+                         "storage_pool",
+                         "tag",
+                         "template",
+                         "user",
+                         "vlan",
+                         "vnet"]}
+    }
+
+    # seed the result dict in the object
+    # we primarily care about changed and state
+    # change is if this module effectively modified the target
+    # state will include any data that you want your module to pass back
+    # for consumption, for example, in a subsequent task
+    result = dict(
```

Comment:
I usually do not place the same comment all over the code where a change is needed. Can you change it here as well?